client/src/pages/Home.js

```diff
@@ -21,18 +21,19 @@ const StyledIntro = styled.div`
   }
 
   @media screen and (min-width: ${mq.tablet.wide.minWidth}) {
-    background-color: ${theme.colors.royalBlue};
-    border-radius: 1rem;
+    background: linear-gradient(337.81deg, #425af2 3.41%, #677cf2 98.66%);
```

Comment:
Awesome, didn't realize it was a gradient. Maybe we should add this to `theme.js` and then access it from there?